migrations/migrations/20180711001908-user_change_password_type.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface
+      .bulkDelete('users', {})
```

Comment:
do you really wanna delete all users here?

Reply:
Didn't see it quite necessary to get into more details of migrating existing users since I need to speed up the training